lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -195,27 +183,32 @@ def prepare_start
       @output_path = File.join(File.dirname(@remote_script), 'output')
       @exit_code_path = File.join(File.dirname(@remote_script), 'exit_code')
       @pid_path = File.join(File.dirname(@remote_script), 'pid')
+      su_method = @user_method.instance_of?(SuUserMethod)
+      wrapper = <<~SCRIPT
+        if [ "$1" == "inner" ]; then
+          echo \$$ > #{@pid_path}
+          (
+            #{@user_method.cli_command_prefix}#{su_method ? "'exec #{@remote_script} < /dev/null '" : "#{@remote_script} < /dev/null"}
+            echo \$? >#{@exit_code_path}
+          ) | tee #{@output_path}
+        else
+          UNSHARE=''
+          if #{UNSHARE_PREFIX} true >/dev/null 2>/dev/null; then
```

Comment:
Does this just test if there is unshare installed and if there isn't, we're happy with just not using it?

Reply:
Correct